package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>rpyutils</name>
+  <version>0.1.0</version>
```

Comment:
Start with 0.0.0 until the first version is tagged.

Reply:
done a5a5e73